### PR TITLE
Migration update wilh nullable column values in Results table

### PR DESCRIPTION
**Pull request by smiechuskce**
Updated migration creating new version of Results table. Null values are now being allowed.

## Motivation and context
#154 

## How has this been tested?
1. Revert migration if applied to the one named "AddedPlainAnswer".
2. Update database.
3. All new records in DB should have null default values.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Views/Admin/Index.cshtml [hunk 1]
```diff
@@ -14,6 +14,7 @@
             <h4>DateTimeUtc - @ViewBag.DateTimeUtc</h4>
             <div>
                 <a href="/Admin/AddTest" class="btn btn-default">Dodaj test</a>
+                <a href="/Admin/CaltulateResults" class="btn btn-default">Oblicz wyniki</a>
```
Comment (WTobor) [inline src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Views/Admin/Index.cshtml]: typo "Calculate"

File: src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Controllers/AdminController.cs [hunk 2]
```diff
@@ -114,6 +115,18 @@ public ActionResult EndTest(int testId)
...
 
+        [HttpPost]
+        public ActionResult CalculateResults(int weekNumber)
+        {
+            if (weekNumber < 1 && weekNumber > 4)
+            {
+                return this.BadRequest("Błędny numer tygodnia.");
+            }
+
+            Process.Start(@"c:\\Windows\\System32\\cmd.exe", weekNumber.ToString(CultureInfo.CurrentCulture.DateTimeFormat));
```
Comment (WTobor) [inline src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Controllers/AdminController.cs]: are you sure this path is correct for the server?
I suggest adding it to configuration ex. in appsettings file

File: src/DevAdventCalendarCompetition/DevAdventCalendarCompetition.TestResultService/BonusPointsRule.cs [hunk 3]
```diff
@@ -0,0 +1,14 @@
+﻿using DevAdventCalendarCompetition.TestResultService.Interfaces;
+
+namespace DevAdventCalendarCompetition.TestResultService
+{
+    public class BonusPointsRule : ITestResultPointsRule
+    {
+        private const int MaxBonus = 30;
```
Comment (WTobor) [inline src/DevAdventCalendarCompetition/DevAdventCalendarCompetition.TestResultService/BonusPointsRule.cs]: it should be readonly

Comment (smiechuskce) [reply src/DevAdventCalendarCompetition/DevAdventCalendarCompetition.TestResultService/BonusPointsRule.cs]: It is a constant integer not a field initialized in a constructor, so I think it should be const.

File: src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Controllers/AdminController.cs (hunk 2 above)
Comment (smiechuskce) [reply src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Controllers/AdminController.cs]: Path is incorrect - Patryk agreed to rewrite the code.

File: src/DevAdventCalendarCompetition/DevAdventCalendarCompetition.TestResultService/BonusPointsRule.cs (hunk 3 above)
Comment (WTobor) [reply src/DevAdventCalendarCompetition/DevAdventCalendarCompetition.TestResultService/BonusPointsRule.cs]: my bad ;p

File: src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Controllers/AdminController.cs (hunk 2 above)
Comment (WTobor) [reply src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Controllers/AdminController.cs]: so we can add a comment here to remember to fix it :)

**Pull request merged**